src/main/bash/sdkman-utils.sh

```diff
@@ -32,7 +32,7 @@ function __sdkman_secure_curl() {
 
 function __sdkman_secure_curl_download() {
 	local curl_params
-	curl_params=('--progress-bar' '--location')
+	curl_params=('--progress-bar' '--location' '--remove-on-error')
```

Comment:
Hi! I would actually challenge my own suggestion 😄! While that would work for newer versions of `curl`, it may break on older versions, no?

Reply:
Ah, and it shows 😆 

`curl: option --remove-on-error: is unknown`